package.json

```diff
@@ -24,6 +24,7 @@
     "prebuild": "rm -rf dist",
     "build-publish": "npm run build-node && npm publish",
     "test": "jest",
+    "test:int": "jest --forceExit --detectOpenHandles",
```

Comment:
Do we really need this test npm script or it was just for testing purposes for this PR?

Reply:
it is needed in case `tx.test.js` is run. Since `withdrawCircuit` functionality is added, zk-proof is computed in background with `snarkjs` and jest does not stop processes unless this flag options are set.